packages/config.ts

```diff
@@ -0,0 +1,3 @@
+export const QrCodeTextGenerator: string =
```

Comment:
Nesses casos de assing direto, o TypeScript consegue inferir o timpo, ou seja, ele sabe que é uma string:

```suggestion
export const QrCodeTextGenerator =
```